examples/BuddyResNet18/import-resnet18.py

```diff
@@ -0,0 +1,56 @@
+# ===- import-resnet18.py ------------------------------------------------------
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ===---------------------------------------------------------------------------
+#
+# This is the test of resnet18 model.
+#
+# ===---------------------------------------------------------------------------
+
+import torch
+import torchvision
+from torch._inductor.decomposition import decompositions as inductor_decomp
+
+from buddy.compiler.frontend import DynamoCompiler
+from buddy.compiler.ops import tosa
+
+
+model = torchvision.models.resnet18()
+model = model.eval()
+
+# Initialize Dynamo Compiler with specific configurations as an importer.
+dynamo_compiler = DynamoCompiler(
+    primary_registry=tosa.ops_registry,
+    aot_autograd_decomposition=inductor_decomp,
+)
+
+data = torch.randn([1, 3, 224, 224])
+right_result = model(data)
+
+with torch.no_grad():
+    model_opt = torch.compile(model, backend=dynamo_compiler)
+    test_result = model_opt(data)
+
+assert torch.allclose(right_result, test_result, atol=1e-5)
+print(right_result)
+print(test_result)
+
+
```

Comment:
When I add `torch._dynamo.reset()` to eliminate the warning:
```
UserWarning: changing options to `torch.compile()` may require calling `torch._dynamo.reset()` to take effect
```
However, the assertion `assert len(graphs) == 1` failed.
```
Traceback (most recent call last):
  File "/home/zhanghongbin/buddy-mlir/examples/BuddyResNet18/import-resnet18.py", line 54, in <module>
    assert len(graphs) == 1
AssertionError
```
The `len(graphs)` will be `2`.

Reply:
It's because, after add dynamo reset, it will compile resnet18 again, so there will be two graph from two compile. We will update test script and fix these warning and errors.